packages/components/src/tooltip/index.module.scss

```diff
@@ -2,27 +2,29 @@
 
 $tooltip-arrow-size: 10px;
 
+.wrap:hover .tooltip {
+  visibility: visible;
+}
+
 .tooltip {
   position: relative;
   cursor: default;
+  visibility: hidden;
```

Comment:
does positioning work with `visibility: hidden`? Didn't popper go crazy if it got initialized with `display: none`?